pkgs/os-specific/linux/kernel/common-config.nix

```
@@ -562,6 +562,12 @@ let
       KEYS_REQUEST_CACHE               = whenAtLeast "5.3" yes;
       # randomized slab caches
       RANDOM_KMALLOC_CACHES            = whenAtLeast "6.6" yes;
+
+      # NIST SP800-90A DRBG modes - enabled by most distributions, required by some kernel modules (ShuffleCake)
```

```suggestion
      # NIST SP800-90A DRBG modes - enabled by most distributions,
      # required by some out-of-tree modules (Shufflecake)
```

Just because if you don't know what Shufflecake is, you're going to assume it's an in-tree module and wonder why its dependencies need to be explicitly enabled.

---

done

---

Now it doesn't mention Shufflecake at all — was that intentional?

---

oops my mistake - forgot to save the file.